app/src/components/BookDetails/BookDetails.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react'
+
+
+import './BookDetails.css'
+
+export default class BookDetails extends Component {
+  constructor( props ) {
+    super( props )
+    this.state = {
+      id: this.props.id,
+      book: {}
+    }
+  }
+
+  componentDidMount() {
+    let fetchURL = `/books/${this.state.id}`
+    fetch( fetchURL )
+     .then( response => response.json() )
+     .then( json => json.data )
+     .then( book => {
+       this.setState({ book: book[0] })
```

Comment:
Why do you need to index into `book`. Seems like it would make more sense for you fetch call to just return what you need.